pkg/cfg/cfg.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 // Version number of the X-Ray daemon.
-const Version = "3.2.0"
+var Version = "unknown"
```

Comment:
The CI is setting the version to the git hash. The main reason to push alpha tag instead of git hash for the docker image is just to reduce Docker Hub storage, but here it doesn't matter so much.

It will provide the release version after creating a release workflow.

I guess unknown or maybe dev are better when CI hasn't provided a value.